en/messages.json

```diff
@@ -318,6 +318,24 @@
             }
         }
     },
```

Comment:
This version is for premium users. 
<img width="157" alt="image" src="https://user-images.githubusercontent.com/13066134/190508584-043466b3-2877-44b8-acd3-584db9ae9327.png">